repos/fdbt-admin/src/pages/Reporting.tsx

```diff
@@ -104,61 +119,71 @@ const Reporting = ({ isFullAdmin }: ReportingProps): ReactElement => {
                     });
                 });
 
-                setRegisteredNocs(nocs);
+                setRegisteredNocs(nocs.sort());
 
                 getExportsFromMatchingDataBucket()
-                    .then((netexData) => setNetexFiles(netexData))
+                    .then((netexData) => {
+                        setThirtyDayNetex(netexFilesGenerated(nocs.sort(), netexData, 30));
```

Comment:
can we create one variable like
const sortedNocs = nocs.sort()
and use this in the line, instead of calling the sort method again ?

Reply:
Sure.